ci-operator/config/openshift/openshift-ansible/release-3.10.json

```diff
@@ -0,0 +1,54 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.10",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_rpm_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.10",
+      "tag": "base"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.9"
+  },
+
+  "rpm_build_commands": "tito tag --offline --accept-auto-changelog --use-release '9999%{?dist}'; tito build --output=\"_output/local/releases\" --rpm --test --offline --quiet; mv _output/local/releases/noarch/* _output/local/releases/",
+  "test_binary_build_commands": "yum install -y gcc libffi-devel python-devel openssl-devel python-pip; pip install tox; chmod uga+w /etc/passwd",
+
+  "images": [
+    {
+      "from": "base",
+      "to": "ansible",
+      "dockerfile_path": "images/installer/Dockerfile"
+    }
+  ],
+
+  "tests": [
+    {
+      "as": "unit",
+      "from": "test-bin",
+      "commands": "echo \"${USER:-default}:x:$(id -u):$(id -g):Default User:$HOME:/sbin/nologin\" >> /etc/passwd; tox 2>&1 | tee /tmp/artifacts/output.log",
```

Comment:
Why didn't you just back port the entry point changes to 3.10?  That would have been a better option.

Reply:
Nm, different issue.